src/wallet/rpcwallet.cpp

```diff
@@ -2660,6 +2664,104 @@ UniValue z_getnewaddress(const UniValue& params, bool fHelp)
     }
 }
 
+UniValue z_getnewaccount(const UniValue& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+    if (fHelp || params.size() > 0)
+        throw runtime_error(
+            "z_getnewaccount\n"
+            "\nSelects the next available sequential account number within the UA-compatible HD seed phrase,"
+            "\ngenerates the corresponding spend authorities for the best and second-best shielded pools, and the transparent pool,"
+            "\nreturns the new account number and the default / first UA containing the default addresses for each pool.\n"
+            "\nResult:\n"
+            "{\n"
+            "  \"account\": n,       (numeric) next available account number\n"
+            "  \"unifiedaddress\"    (string) The default address for this account\n"
+            "}\n"
+            "\nExamples:\n"
+            + HelpExampleCli("z_getnewaccount", "")
+            + HelpExampleRpc("z_getnewaccount", "")
+        );
+
+    if (!fExperimentalOrchardWallet) {
+        throw JSONRPCError(RPC_WALLET_ENCRYPTION_FAILED, "Error: the Orchard wallet experimental extensions are disabled.");
+    }
+    int64_t account = 999999; // placeholder
+    UniValue result(UniValue::VOBJ);
+    result.pushKV("account", account);
+    result.pushKV("unifiedaddress", "stubOrchardAddr");
+    return result;
+}
+
+UniValue z_getaddressforaccount(const UniValue& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+    if (fHelp || params.size() < 1 || params.size() > 3)
+        throw runtime_error(
+            "z_getaddressforaccount account ( diversifier_index ) ( [\"protocol\", ...] )\n"
```

Comment:
```suggestion
            "z_getaddressforaccount account (( diversifier_index ) [\"protocol\", ...] )\n"
```

Reply:
"protocol" here should be consistent with what we decide for the key name in the return value.

Reply:
@daira I don't understand this comment. What key in the return value are you referring to?